include/ck/tensor_operation/gpu/device/impl/device_grouped_gemm_multiple_d_dl.hpp

```diff
@@ -802,8 +804,16 @@ struct DeviceGroupedGemmMultipleD_Dl : public DeviceGroupedGemm<ALayout,
         return this->SetWorkSpacePointer(p_arg, p_dev_kernel_args);
     }
 
-    // Use for the async host args copy flow
-    void SetHostKernelArgs(BaseArgument* p_arg, void* p_host_kernel_args) const
+    //----------------------------------------------------------------------------------------------
+    /// @brief      Sets the host kernel arguments pointer and copies that data on the host side.
+    ///             This function can be utilised to use pinned memory for the host args and
+    ///             achieve fully async data copy.
+    ///
+    /// @param      p_arg              The pointer to the Argument we're going to update.
+    /// @param[in]  p_host_kernel_args The pointer to the host memory which contains kernel
+    ///                                arguments.
```

Comment:
That's not exactly true. The memory pointed by this pointer rather do not **yet** have those arguments. We will copy them there.